script.js

```diff
@@ -0,0 +1,18 @@
+const hamburger = document.querySelector('.fa-bars');
+const MBMenu = document.querySelector('.mb_menu');
```

Comment:
[OPTIONAL] 
👌👌
- [ ] It would be greatly appreciated if you could utilize meaningful variable names throughout the codebase.

- [ ]  Choosing descriptive and clear variable names, will enhance the readability and maintainability of the code. ✔️